components/LineEdit.qml

```diff
@@ -210,6 +239,27 @@ Item {
             onTextChanged: item.textUpdated()
             topPadding: 10
             bottomPadding: 10
+            echoMode: isPasswordHidden() ? TextInput.Password : TextInput.Normal
+
+            MoneroComponents.Label {
+                visible: password || passwordLinked
+                fontSize: eyeMouseArea.containsMouse ? 24 : 20
```

Comment:
```suggestion
                fontSize: eyeMouseArea.containsMouse ? 22 : 20
```

24 feels a bit too much

Reply:
Is a copy-paste of the current code.

TBH, I don't get why it gets bigger on hover. Would prefer to drop this. Thoughts?

Reply:
Yes, I prefer dropping this and only changing color.

Reply:
Updated